### PR TITLE
Osx support

### PR DESCRIPTION
**Pull request by traviswpeters**
- Updated setup.py to install LightAquaBlue framework (for bluetooth use on osx) and to install the pybluez/lightblue projects correctly.
- Add basic support to osx.py; specifically, discover_devices() works (try inquiry.py). Other code is in progress and hasn't yet been verified.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,8 +5,22 @@
 import platform
 import os
 
+WIN32 = sys.platform.startswith("win32")
```
Comment (karulis) [inline setup.py:8]: I would just define here empty dict:
pack_dir = dict()
then remove lines 8-11 and 14-23, after line 92 add
pack_dir = { 'lightblue': "osx" }
and then change line 134 to
package_dir = pack_dir

I think that should have same functionality with less code

**Comment by traviswpeters** [comment]
Sorry I haven’t responded in a bit — my internship ends next week and I’ve been hittin it pretty hard to try to wrap stuff up nicely before I take off!

I hope all is well with you! :)

-Travis

**Comment by karulis** [comment]
No worries Travis:)
I was going to send You mail that I have bought Mac Mini with os x 10.10 yesterday:)
So we can work on port to OSX together.

But focus on internship and after it ends we can continue with porting:)
 
Piotrek

**Comment by traviswpeters** [comment]
Thanks for being so understanding!

And that is awesome that you bought a Mac mini -- is this your first Mac?! If so, welcome to the dark side :) Hahahaha 

-Travis

**Pull request merged**